### PR TITLE
0.10.0 Docs

### PR DESCRIPTION
**Pull request by Lagoja**
## Summary

Update docs and examples for Devbox 0.10.0

## How was it tested?

Localhost + Vercel. Also ran lint

### CODE REVIEW

**Review by mikeland73 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,4 @@ __pycache__/
 
 # deployment
 .vercel
+examples/development/nodejs/nodejs-yarn/.yarn/install-state.gz
```
Comment (LucilleH) [inline .gitignore:40]:
```suggestion
.yarn
```

File: docs/app/docs/faq.md [hunk 2]
```diff
@@ -31,14 +31,33 @@ Yes! Devbox can be installed on any Linux distribution, including NixOS. You can
...
 
 ## I'm trying to build a project, but it says that I'm missing `libstdc++`. How do I install this library in my project?
 
 This message means that your project requires an implementation of the C++ Standard Library installed and linked within your shell. You can add the libstdc++ libraries and object files using `devbox add stdenv.cc.cc.lib`. 
 
+## I'm seeing a ``GLIBC_X.XX' not found` error when I try to install my packages, or when I install packages from PyPi/RubyGems/NPM/Cargo/other package manager in my shell
+
+This message usually occurs when using older packages, or when mixing different versions of packages within a single shell. The error tends to occur because each Nix package comes bundled with all of it's dependencies, including a version of the C Standard Library, to ensure reproducibility. If your interpreter (Python/Ruby/Node) or runtime is using an older version of `glibc` than what your other packages expect, they will throw this error. 
+
+There are two ways to work around this issue:
```
Comment (LucilleH) [inline docs/app/docs/faq.md:56]:
```suggestion
There are three ways to work around this issue: 
```

File: docs/app/docs/faq.md [hunk 3]
```diff
@@ -31,14 +31,33 @@ Yes! Devbox can be installed on any Linux distribution, including NixOS. You can
...
+
+This message usually occurs when using older packages, or when mixing different versions of packages within a single shell. The error tends to occur because each Nix package comes bundled with all of it's dependencies, including a version of the C Standard Library, to ensure reproducibility. If your interpreter (Python/Ruby/Node) or runtime is using an older version of `glibc` than what your other packages expect, they will throw this error. 
+
+There are two ways to work around this issue: 
+1. You can update your packages to use a newer version (using `devbox add`). This newer version will likely come bundled with a newer version of `glibc`. 
+2. You can use `devbox update` to get the latest Nix derivation for your package. Newer derivations may come bundled with newer dependencies, including `glibc`
+3. If you need to use an exact package version, but you still see this error, you can patch it to use a newer version of glibc using `devbox add <package>@<version> --patch-glibc`. This will patch your package to use a newer version of glibc, which should resolve any incompatibility issues you might be seeing. This patch will only affect packages on Linux.
```
Comment (LucilleH) [inline docs/app/docs/faq.md:59]: Should we bold or highlight `This patch will only affect packages on Linux`?

File: docs/app/docs/guides/creating_plugins.md [hunk 4]
```diff
@@ -126,6 +139,56 @@ A single `bash` command or list of `bash` commands that should run before the us
...
+            ]
+        }
+    }
+}
+``` 
+Scripts defined in a plugin will be overridden if a user's `devbox.json` defines a script with the same name. For example, if both the plugin and the devbox.json that includes it defined a `print_once` script, the version in the user's `devbox.json` will take precedence in the shell. 
+
+#### `include` *string[]*
+
+Includes can be used to explicitly add extra configuration from [plugins](../guides/plugins.md) to your Devbox project. Plugins are parsed and merged in the order they are listed.
```
Comment (LucilleH) [inline docs/app/docs/guides/creating_plugins.md:176]:
```suggestion
Include can be used to explicitly add extra configuration from [plugins](../guides/plugins.md) to your Devbox project. Plugins are parsed and merged in the order they are listed. 
```

Comment (LucilleH) [reply docs/app/docs/guides/creating_plugins.md:176]: includes or include???

**Review by LucilleH — APPROVED** [review]

File: docs/app/docs/faq.md (hunk 2 above)
Comment (LucilleH) [reply docs/app/docs/faq.md]: three!!! @Lagoja you did not change it

**Pull request merged**